amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -209,14 +236,41 @@ vx_status Resize_Register(vx_context);
 #define VX_KERNEL_RPP_CROPPD_NAME   							"org.rpp.CropPD"
```

Comment:
@fiona-gladwin Please check indentation here and below

Reply:
@LakshmiKumar23 I am not able to see any indentation issues when I check in VScode please let me know which line the issue is

Reply:
@fiona-gladwin Please open the file on git and check. It might be space vs tab and looks same on vs code

Reply:
In git file also I am unable to see the issue

Reply:
@fiona-gladwin. Refer below. It's all over the place
![image](https://github.com/LakshmiKumar23/MIVisionX/assets/27569995/4fbce97b-59ee-40b3-bc39-2f4a258a281b)

Reply:
Done. Please check